ETC.tsv

```diff
@@ -3,9 +3,9 @@ ETC_20150317_000002	$The Kepa is not a strong monster. However, attacking it rec
 ETC_20150317_000003	$Large Kepa
 ETC_20150317_000004	$There are people who try to eat the Large Kepa sometimes. Don't let them.
 ETC_20150317_000005	$Red Kepa
-ETC_20150317_000006	$Contrary to common awareness, it is Red Kepa which is more similar to the original Kepa
-ETC_20150317_000007	$Poisoned Kepa.
-ETC_20150317_000008	$Monster
+ETC_20150317_000006	$Contrary to common belief, it's Red Kepa that are more similar to the original Kepa.
```

Comment:
Oh oh, I see.  In this case, I feel like referring to them as Kepas is definitely a better way for indicating them being plural, and I'm pretty sure the usage of "Kepas" is much more common in various parts of the translation already, so I'll just put it back to that for simplicity's sake.  But yeah, we need to figure it out.